docker/Dockerfile

```diff
@@ -20,8 +20,6 @@ RUN pip install --no-cache-dir --upgrade pip && pip install --no-cache-dir --use
 # COMMON IMAGE --------------------------------------
 # ---------------------------------------------------
 FROM python:3.8-alpine3.11 as common
-# bash is needed for ./start/sh script
-RUN apk add --update --no-cache bash
 # copy libraries from build stage
 COPY --from=BuildStage /root/.local /root/.local
 # copy wait-for-it script
```

Comment:
I'm running this command as test plan to see nothing breaks:
```
docker compose -f docker/docker-compose-example.yml up
```
i get this error:
```
opal_client_1        | sh: /usr/wait-for.sh: not found
opal_client_1 exited with code 127
```
i guess you need to also need to copy the wait-for script into the container - looks like the container expects the script to sit at `/usr/wait-for.sh`

Reply:
Sorry, I missed that part and didn't notice I can test it with docker-compose locally. Now it's fixed.